tests/Microsoft.Bot.Builder.Adapters.WeChat.Tests/Microsoft.Bot.Builder.Adapters.WeChat.Tests.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Migrate the tests to use XUnit instead of MSTest, we are standardizing on XUnit for new projects
Move the test project inside a Adapters folder (create a new folder and put it there, we have more adapters coming and this will help us keep things organized).

Reply:
convert to XUnit and move to Adapters folder